pkg/blackduck/ctl_blackduck.go

```diff
@@ -213,6 +228,8 @@ func (ctl *Ctl) AddSpecFlags(cmd *cobra.Command, master bool) {
 	cmd.Flags().StringVar(&ctl.AuthCustomCAFilePath, "auth-custom-ca-file-path", ctl.AuthCustomCAFilePath, "Absolute path to a file for the Custom Auth CA for Black Duck")
 	cmd.Flags().StringVar(&ctl.Type, "type", ctl.Type, "Type of Black Duck")
 	cmd.Flags().StringVar(&ctl.DesiredState, "desired-state", ctl.DesiredState, "Desired state of Black Duck")
+	cmd.Flags().BoolVar(&ctl.DbMigrateState, "state-db-migrate", ctl.DbMigrateState, "Create Black Duck in the database-migration state")
+	cmd.Flags().BoolVar(&ctl.StopState, "state-stop", ctl.StopState, "Create Black Duck in the stop state")
```

Comment:
We don't want a stop option, as we have start/stop commands for that.

Reply:
@rrati if we don't add a stop flag, then currently a user would be able to create in migration mode but not in stopped mode.

Reply:
Let's leave off the stop option on creation for now